### DISCUSSION TITLE
justify-around isnot working in tailwindcss

### DISCUSSION THREAD

**Discussion by SheikhElMoctarG** [tailwindlabs/tailwindcss#12015] [Help]
**What version of Tailwind CSS are you using?**
v3.3.3
For example: v2.0.4

For example: postcss-cli 8.3.1, Next.js 10.0.9, webpack 5.28.0

**What browser are you using?**
edge
For example: Chrome, Safari, or N/A

**What operating system are you using?**

, Windows

--------------
I tried to use class justify-around, but it isnot working, so I needed to add it as style.
please fix it

**Comment by basavarajakj** [comment]
Could you please provide code snippet.

**Comment by RobinMalfait** [comment]
Hey!

Checkout https://tailwindcss.com/docs/justify-content#space-around on how the `justify-around` works.
